payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncherActivity.kt

```diff
@@ -0,0 +1,178 @@
+package com.stripe.android.googlepaylauncher
+
+import android.content.Intent
+import android.content.pm.ActivityInfo
+import android.os.Build
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.os.bundleOf
+import androidx.lifecycle.lifecycleScope
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.wallet.AutoResolveHelper
+import com.google.android.gms.wallet.PaymentData
+import com.stripe.android.model.PaymentMethod
+import kotlinx.coroutines.launch
+
+/**
+ * [GooglePayPaymentMethodLauncherActivity] is used to return the result of a Google Pay operation.
+ * The activity's result will be a [GooglePayPaymentMethodLauncher.Result].
+ *
+ * [GooglePayPaymentMethodLauncherActivity] will:
+ * 1. Verify that Google Pay is available
+ * 2. Create a [PaymentDataRequest](https://developers.google.com/pay/api/web/reference/request-objects#PaymentDataRequest)
+ * 3. Create a [PaymentMethod] using the Google Pay token
+ *
+ * Use [GooglePayPaymentMethodLauncherContract] to start [GooglePayPaymentMethodLauncherActivity].
+ *
+ * See [Troubleshooting](https://developers.google.com/pay/api/android/support/troubleshooting)
+ * for a guide to troubleshooting Google Pay issues.
+ */
+internal class GooglePayPaymentMethodLauncherActivity : AppCompatActivity() {
```

Comment:
I am probably getting reviews mixed up.  Is this one going to remain internal?

Reply:
This is the `GooglePayLauncher` variant for just getting a `PaymentMethod`